pkgs/by-name/sa/salmon/package.nix

```
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, autoreconfHook
+, bash
+, boost
+, bzip2
+, cereal_1_3_2
+, cmake
+, curl
+, fetchFromGitHub
+, gcc
+, jemalloc
+, lbzip2
+, libgff
+, libiconv
+, libstaden-read
+, lzlib
+, pkg-config
+, tbb_2021_8
+, xz
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "salmon";
+  version = "1.10.2";
+
+  pufferFishSrc = fetchFromGitHub {
+    owner = "COMBINE-lab";
+    repo = "pufferfish";
+    rev = "salmon-v${version}";
+    sha256 = "sha256-JKbUFBEsqnENl4vFqve1FCd4TI3n9bRi2RNHC8QGQGc=";
+  };
+
+  src = fetchFromGitHub {
+    owner = "COMBINE-lab";
+    repo = "salmon";
+    rev = "v${version}";
+    sha256 = "sha256-kwqoUmVCqjr/xRxJjQKaFjjCQW+MFASHJ2f9OiAumNU=";
+  };
+
+  patches = [
+    # Use pufferfish source fetched by nix
+    ./fetch-pufferfish.patch
+  ];
+
+  postPatch = "patchShebangs .";
+
+  buildInputs = [
+    zlib
+    lzlib
+    lbzip2
+    xz
+    bzip2
+    cereal_1_3_2
+    tbb_2021_8
+    libgff
+    libstaden-read
+    jemalloc
+    boost
+    curl
+  ] ++ lib.optionals stdenv.isDarwin [ libiconv ];
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  strictDeps = true;
+
+  meta = {
+    description = "Tool for quantifying the expression of transcripts " +
+                  "using RNA-seq data";
```

Just one line please.

```suggestion
    description = "Tool for quantifying the expression of transcripts using RNA-seq data";
```

---

Yeh, I'm a bit obsessed with trying to keep it within 80 characters. I tried it in another way :)